### ISSUE TITLE
repr(six.moves) throws an AttributeError for cpython 3.12

### ISSUE THREAD

**Issue by 15r10nk** [benjaminp/six#376]
```python
import six.moves

repr(six.moves)
```

output (Python 3.12.0rc3+):
```python
Traceback (most recent call last):
  File "/home/frank/projects/executing/bug.py", line 3, in <module>
    repr(six.moves)
  File "<frozen importlib._bootstrap>", line 545, in _module_repr
  File "<frozen importlib._bootstrap>", line 830, in _module_repr_from_spec
AttributeError: '_SixMetaPathImporter' object has no attribute '_path'
```

**Comment by lauromoura** [comment]
Looks like the cause is CPython's https://github.com/python/cpython/commit/bd221c01ddcd2cad102133611fab3569a99680b0, which expects the spec to have the `_path` parameter, although this attribute is not mentioned explicitly in the [PEP302](https://peps.python.org/pep-0302/) (Import hooks PEP).

Not sure if the right fix would be there or adding a `_path` attribute to six's meta importer.

**Comment by 15r10nk** [comment]
looks to me like this should be fixed in cpython. It should also be a problem for any importer, or is there a reason why they should have a _path attribute?

**Comment by benjaminp** [comment]
Did anyone file a CPython bug?

**Comment by 15r10nk** [comment]
I did not. but I am having trouble making a repro without six. I would be nice if someone else could submit a bug report.

**Comment by hmstepanek** [comment]
I was running into this issue on 3.12.0 but it looks like this has been fixed on 3.12.1: https://github.com/python/cpython/issues/112414.
